### DISCUSSION TITLE
Can we set Job's StringVar with password properity?

### DISCUSSION THREAD

**Discussion by alicechiou** [nautobot/nautobot#5587] [Q&A]
Hi teams, 
Is there any property on Job's StringVar by which we can hide user input information on the UI?
just like html <input type="password">
We don't want sensitive information to be displayed in the user interface.
![image](https://github.com/nautobot/nautobot/assets/39262687/3b539090-9b26-4039-8cc0-25dc41f0ea5f)

**Comment by glennmatthews** [comment]
Ideally you'd use the Secrets feature for this sort of information, rather than providing credentials as plaintext Job inputs. There are numerous places where Job parameters are potentially stored in the database and/or exposed to other users, for example in scheduled jobs, job results, etc.

**Comment by alicechiou** [comment]
But can we set different secret by different user? I mean when different user execute the job, they can only choose their own secret and use the secret connect to their own device.
